x/claim/keeper/hooks.go

```diff
@@ -0,0 +1,81 @@
+package keeper
+
+import (
+	"github.com/arkeonetwork/arkeo/x/claim/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+// Hooks wrapper struct
+type Hooks struct {
+	k Keeper
+}
+
+// Return the wrapper struct
+func (k Keeper) Hooks() Hooks {
+	return Hooks{k}
+}
+
+var _ govtypes.GovHooks = Hooks{}
+var _ stakingtypes.StakingHooks = Hooks{}
+
+func (k Keeper) AfterProposalVote(ctx sdk.Context, proposalID uint64, voterAddr sdk.AccAddress) {
```

Comment:
can we please move these two methods to `keeper.go`?

Reply:
done